docs/BUILD-ACCELERATION.md

```
@@ -0,0 +1,141 @@
+已使用 `-DskipTests` 时，测试已跳过。若某处仍会跑测试，可再加：
+
+```bash
+-Dmaven.test.skip=true
+```
+
+---
+
+## 建议的日常组合
+
+| 场景 | 命令示例 |
+|------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)